.github/workflows/rust-skipped.yml

```
@@ -0,0 +1,30 @@
+name: Rust
+
+on:
+  pull_request:
+    path-ignored: # Make sure to keep sync'd https://github.com/pyrsia/pyrsia/blob/main/.github/workflows/rust.yml#L8
```

isn't the correct syntax `paths-ignored`?

---

is the correct syntax `paths-ignore`?

---

#749

---

yes, there are 2 workflows.  The regular one for running the actual build (rust.yaml) which uses `path:`  and the converse (rust-skipped.yaml) which uses the `path-ignore:`  to make sure the required checks are covered.  This setup keeps PRs from being hung up on required checks when no Rust files have changed.

---

ok, so it should be:
```suggestion
    paths-ignore: # Make sure to keep sync'd https://github.com/pyrsia/pyrsia/blob/main/.github/workflows/rust.yml#L8
```
?